packages/react/primitive/src/Primitive.tsx

```diff
@@ -10,27 +9,14 @@ import type * as Polymorphic from '@radix-ui/react-polymorphic';
 const NAME = 'Primitive';
 const DEFAULT_TAG = 'div';
 
-type PrimitiveOwnProps = {
-  /**
-   * A string to use as the component selector for CSS purposes. It will be added as
-   * a data attribute. Pass `null` to remove selector.
-   *
-   * @defaultValue radix-{component-name}
-   *
-   * @example
-   * `AccordionItem` component will have `radix-accordion-item` selector by
-   * default. Passing `selector="org-accordion-item"` will replace selector
-   * with `org-accordion-item`.
-   */
-  selector?: string | null;
-};
+type PrimitiveOwnProps = {};
```

Comment:
I wasn't sure here if we normally keep it as empty or just don't have it?

Reply:
Not really sure what we do tbh but I'm easy either way haha